build/pxc-entrypoint.sh

```diff
@@ -454,6 +456,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			else
 				echo >&2 "WSREP: Failed to recover position: "
 				cat "$wsrep_verbose_logfile"
```

Comment:
why we need two `cat` commands, one one line 444 and second in 458

Reply:
yes, we can remove this `cat`